go.sum

```diff
@@ -1,2 +1,42 @@
+github.com/Microsoft/go-winio v0.4.14 h1:+hMXMk01us9KgxGb7ftKQt2Xpf5hH/yky+TDA+qxleU=
```

Comment:
do we really required all these deps? sound like a NodeJS app 😄

Reply:
yeah we do. Come on, it's just one file, check some node_modules out there, there is not even comparison hahaha

Reply:
fair enough